### ISSUE TITLE
No binaries in package from npm (0.0.5)

### ISSUE THREAD

**Issue by AntonTimiskov** [lithin/serverless-run-function-plugin#7]
Last package update broke the serverless import.
After `npm install -g serverless-run-function-plugin` I got:

```
/usr/local/lib/node_modules/serverless-run-function-plugin# ls -al
total 16
drwxr-xr-x 2 nobody root 4096 Nov  1 13:01 .
drwxrwxr-x 8    500  500 4096 Nov  1 13:01 ..
-rw-r--r-- 1 nobody root  738 Nov  1 11:29 README.md
-rw-r--r-- 1 root   root 2174 Nov  1 13:01 package.json
/usr/local/lib/node_modules/serverless-run-function-plugin# cat package.json 
{
  "name": "serverless-run-function-plugin",
  "version": "0.0.5",
  "description": "Serverless plugin to run a function",
  "main": "./dist/index.js",
  "files": [
    "dist"
  ],
```

**Comment by lithin** [comment]
Ohh so sorry about that, fix is on the way

**Comment by lithin** [comment]
@AntonTimiskov it's fixed in version 0.0.6 :)
